<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="kc-spartans" ignored="0" name="Test" tests="2" failures="1" timestamp="2020-11-23T13:11:10 UTC" time="27.122" errors="0">
  <testcase name="testBadRequest" time="15.327" classname="Hackathon_Test.APITest"/>
  <testcase name="testGoodRequest" time="27.008" classname="Hackathon_Test.APITest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Hackathon_Test.APITest.printlog(APITest.java:798)
at Hackathon_Test.APITest.apiTestPASS(APITest.java:514)
at Hackathon_Test.APITest.testGoodRequest(APITest.java:281)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testGoodRequest -->
</testsuite> <!-- Test -->
